frontend/src/locales/en.yml

```diff
@@ -224,7 +224,8 @@ task:
     warn: Warning
     error: Error
   earliest-allowed-time-no-modify: >-
-    You can not modify this field because you are not the issue assignee or it's not pending to run.
+    You can not modify this field because you are not the issue assignee or it's
```

Comment:
@xiaoluoboding @LiuJi-Jim could you please try to figure out why it's been reformatted again?

Reply:
Sorry, I didn't notice this, it seems to be automatically formatted by the plugin.